SampleProject/VOKCoreDataManagerTests/BundleTests.m

```diff
@@ -0,0 +1,30 @@
+//
+//  BundleTests.m
+//  SampleProject
+//
+//  Created by Ellen Shapiro (Vokal) on 10/20/15.
+//
+//
+
+#import <XCTest/XCTest.h>
+#import "VOKCoreDataManager.h"
+
+@interface BundleTests : XCTestCase
+
+@end
+
+@implementation BundleTests
+
+- (void)testWorksWithoutSettingCustomBundle
+{
+    XCTAssertNoThrow([[VOKCoreDataManager sharedInstance] setResource:@"VICoreDataModel" database:nil]);
+}
+
+- (void)testWorksWithSettingCustomBundle
+{
+    XCTAssertNoThrow([[VOKCoreDataManager sharedInstance] setResource:@"VICoreDataModel"
```

Comment:
I still don't have the faintest fucking clue why this wasn't working earlier, but it's working now.

NOTE: I did have to update the bundle it was using to the main bundle so that the MOM didn't have to be embedded in the test bundle. @vokal-isaac in theory this shouldn't break anything major, but in practice....should I change this to a major release?